ocpp/v16/enums.py

```diff
@@ -619,3 +631,156 @@ class ValueFormat(str, Enum):
+    """
+    accepted = "Accepted"
+    rejected = "Rejected"
+
+
+class CertificateStatus(str, Enum):
+    """
+    CertificateStatusEnumType is used by: InstallCertificate.conf
+    """
+    accepted = "Accepted"
+    rejected = "Rejected"
+    failed = "Failed"
+
+
+class CertificateUse(str, Enum):
+    """
+    CertificateUseEnumType is used by: GetInstalledCertificateIds.req,
+    InstallCertificate.req
+    """
+    central_system_root_certificate = "CentralSystemRootCertificate"
+    manufacturer_root_certificate = "ManufacturerRootCertificate"
+
+
+class DeleteCertificateStatus(str, Enum):
+    """
+    DeleteCertificateStatusEnumType is used by: DeleteCertificate.conf
+    """
+    accepted = "Accepted"
+    failed = "Failed"
+    not_found = "NotFound"
+
+
+class SignedFirmwareStatus(str, Enum):
+    """
+    FirmwareStatusEnumType is used by: SignedFirmwareStatusNotification.req
+    """
+    certificate_verified = "CertificateVerified"
+    downloaded = "Downloaded"
+    download_failed = "DownloadFailed"
+    downloading = "Downloading"
+    download_scheduled = "DownloadScheduled"
+    download_paused = "DownloadPaused"
+    idle = "Idle"
+    installation_failed = "InstallationFailed"
+    installing = "Installing"
+    installed = "Installed"
+    install_rebooting = "InstallRebooting"
+    install_scheduled = "InstallScheduled"
+    install_verification_failed = "InstallVerificationFailed"
+    invalid_signature = "InvalidSignature"
+    invalid_certificate = "InvalidCertificate"
+    revoked_certificate = "RevokedCertificate"
+    signature_verified = "SignatureVerified"
+
+
+class GenericStatus(str, Enum):
+    """
+    Generic message response status
+    """
+    accepted = "Accepted"
+    rejected = "Rejected"
+
+
+class GetInstalledCertificateStatus(str, Enum):
+    """
+    GetInstalledCertificateStatusEnumType is used by:
+    GetInstalledCertificateIds.conf
+    """
+    accepted = "Accepted"
+    not_found = "NotFound"
+
+
+class HashAlgorithm(str, Enum):
+    """
+    HashAlgorithmEnumType is used by: CertificateHashDataType
+    """
+    sha256 = "SHA256"
+    sha384 = "SHA384"
+    sha512 = "SHA512"
+
+
+class Log(str, Enum):
+    """
+    LogEnumType is used by GetLog.req
+    """
+    diagnostics_log = "DiagnosticsLog"
+    security_log = "SecurityLog"
+
+
+class LogStatus(str, Enum):
+    """
+    LogStatusEnumType is used by: GetLog.conf
+    """
+
+    accepted = "Accepted"
+    rejected = "Rejected"
+    accepted_canceled = "AcceptedCanceled"
+
+
+class ExtendedMessageTrigger(str, Enum):
+    """
+    MessageTriggerEnumType is used by: ExtendedTriggerMessage.req
+    """
+
+    boot_notification = "BootNotification"
+    log_status_notification = "LogStatusNotification"
+    firmware_status_notification = "FirmwareStatusNotification"
+    heartbeat = "Heartbeat"
+    meter_values = "MeterValues"
+    sign_charge_point_certificate = "SignChargePointCertificate"
+    status_notification = "StatusNotification"
```

Comment:
I dont think we need to create a new Status for the Extended Trigger. We can just add the extra fields that are added by the whitepaper and add a comment mentioning it. This will then be in accordance with the Whitepaper which also doesnt change the name of the enum "TriggerMessageStatusEnumType"

Reply:
I'll make that change also.